java/test/jmri/jmrix/can/cbus/CbusLightManagerTest.java

```diff
@@ -328,7 +328,7 @@ public void testProvideswhenNotNull() {
     @Test
     public void testcreateNewLightException() {
         Assert.assertThrows(IllegalArgumentException.class, () -> ((CbusLightManager)l).createNewLight("",null));
-        JUnitAppender.assertErrorMessageStartsWith("System name must start with \"ML\"");
+        JUnitAppender.assertErrorMessageStartsWith("Unable to create CbusLight, System name must start with \"ML\"");
```

Comment:
Is it still true that CBUS assumes the system prefix is always "M"?

Reply:
Thanks for looking over Bob.

All of CBUS system prefixes are memo fetched afaik.

In this case, the prefix letter is supplied by a standard CanSystemConnectionMemo default.

If the assert is prepended by `Assertions.assertTrue(memo.setSystemPrefix("L"));`  , 
the exception text changes as expected.

Reply:
@icklesteve Thank you!

Reply:
That being said, I'm having issues with multiple CBUS connections.

Some of the CBUS classes ( eg. CbusNodeTableDataModel, CbusEventTabl;eDataModel ) , use objects of a different memo according to whichever was created in the InstanceManager first.

These really need to be memo fetched based rather than InstanceManager.
I've been developing a PR which resolves this, however still in local testing.